gatorgrade/output/output_functions.py

```diff
@@ -1,2 +1,31 @@
 # This class is used for storing the main functions requested from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output, description output, and colorization of text.
\ No newline at end of file
+# For instance, functions dealing with percentage output, description output, and colorization of text.
+
+import gator
+import os
+import output_tools
+
+# Commands are received as list of tuples.
+# Each tuple in the list is a string and list of strings <string, [strings]>
+# The string is the path to the file or filename and the list is the commands for the file.
+def receive_command(command_info):
+    results = []
+
+    for file_name, commands in command_info:
+        split_commands = []
+        results.append((file_name, []))
+
+        for pre_command in commands:
+

```

Comment:
Would you mind removing some of the excess empty space in this function? Spacing is good for readability of code, but on the other hand too much space can prohibit that.

Reply:
I tried to remove some of the spacing that seemed excessive in the latest commit, do you think they look alright currently?

Reply:
Yes! I think that looks a lot better. Thank you!